app/Http/Controllers/UserController.php

```diff
@@ -84,4 +95,60 @@ public function approval(User $user)
         
         return ['message' => "User has been " . $message];
     }
+
+    public function store(Request $request)
+    {
+        $branches = json_decode($request->branches);
+
+        $this->validate($request, [
+            'name' => 'required',
+            'email' => 'required',
+            'address' => 'required',
+            'phone' => 'required'
+        ]);
+
+        $check_email = User::where('email' , $request->email)->get();
+        if($check_email->isEmpty()) {
+            $password = Str::random(8);
+            $user = new User;
+            $user->name = $request->name;
+            $user->email = $request->email;
+            $user->password = Hash::make($password);
+            $user->address = $request->address;
+            $user->address_2 = $request->address_2;
+            $user->phone = $request->phone;
+            $user->state = $request->state;
+            $user->postcode = $request->postcode;
+            $user->country = $request->country;
+            $user->save();
+
+            foreach($branches as $branch) {
```

Comment:
Looks like this will return error? If I am not mistaken your branches here are just branch ID